great-idea/css/index.css

```diff
@@ -63,4 +63,118 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/*------------------------------------------------------------*/
+
+* {
+	box-sizing: border-box;
+}
+
+body {
+	background: ;
```

Comment:
Did you want to specify a background? Looks like a typo, just be aware of these things